shell/browser/extensions/electron_extension_loader.cc

```diff
@@ -55,10 +66,20 @@ std::pair<scoped_refptr<const Extension>, std::string> LoadUnpacked(
   std::string warnings;
   // Log warnings.
   if (!extension->install_warnings().empty()) {
-    warnings += "Warnings loading extension at " +
-                base::UTF16ToUTF8(extension_dir.LossyDisplayName()) + ":\n";
     for (const auto& warning : extension->install_warnings()) {
-      warnings += "  " + warning.message + "\n";
+      // filter kUnrecognizedManifestKey error. This error does not have any
+      // impact e.g: Unrecognized manifest key 'minimum_chrome_version' etc.
+      std::string check_error = ErrorUtils::FormatErrorMessage(
+          manifest_errors::kUnrecognizedManifestKey, warning.key);
+      if (warning.message != check_error) {
+        warnings += "  " + warning.message + "\n";
+      }
+    }
+
+    if (warnings != "") {
+      warnings = "Warnings loading extension at " +
+                 base::UTF16ToUTF8(extension_dir.LossyDisplayName()) + ":\n" +
+                 warnings;
```

Comment:
I'd prefer to still have these messages accessible. I think changing them to log using `LOG(WARNING)` would be fine. Then they'd only appear if the user opts-in to logging with `--enable-logging`.

e.g.
```cpp
LOG(WARNING) << "Warning loading extension at  " + base::UTF16ToUTF8(extension_dir.LossyDisplayName()) + ": " << warning.message;
```

Additionally, if we want to prevent these messages from appearing, we can add support for the unrecognized manifest keys by modifying https://github.com/electron/electron/blob/main/shell/common/extensions/api/_manifest_features.json and possibly copying some configurations from Chrome.

Reply:
cool. I will submit a PR soon, using LOG(WARNING) to replace the previous way

But adding extra keys to xx and having to implement specific logic can be a lot of work.

I may have a hard time doing it in a short time (because I haven't learned C++ yet...)

Reply:
> But adding extra keys to xx and having to implement specific logic can be a lot of work.

Either with or without the new keys should be fine within the scope of this PR.

> I may have a hard time doing it in a short time (because I haven't learned C++ yet...)

No worries! Do what you think is best and we'll help steer the code in the right direction if it needs changing. 🙂 https://cs.chromium.org is a great way to find code samples within Chromium if you haven't come across it already.

Reply:
I'm talking to the chromium team about the possibility of adding new flags to chromium to do this. If there is any progress, I will give feedback here

Reply:
@samuelmaddock 
Since I am currently very busy at work, I may not have a lot of energy to work on this in the short term.

So my personal opinion is to change it to `LOG(WARNING)` way first, and I will add `TODO comment` to describe this.

I will be happy to implement these api later if I have time

Reply:
This PR will be done with `LOG(WARNING)` first, and I will follow your suggestion to submit a new PR when I have time later